src/applications/static-pages/health-care-manage-benefits/schedule-view-va-appointments-page/tests/index.cypress.spec.js

```diff
@@ -0,0 +1,73 @@
+import cernerUser from '../../fixtures/user/cerner.json';
+import notCernerUser from '../../fixtures/user/notCerner.json';
+
+const TEST_URL = '/health-care/schedule-view-va-appointments/';
+
+const setup = ({ authenticated, isCerner } = {}) => {
+  cy.fixture(
+    '../../src/applications/static-pages/health-care-manage-benefits/fixtures/feature-toggles/enabled.json',
+  ).then(features => {
```

Comment:
Just wanted to point out that we have a [custom command to deal with fixtures](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-migration-guide.md#test-data-fixtures-) (`cy.syncFixtures`) more cleanly.